components/brave_ads/core/internal/account/user_data/fixed/country_code_user_data.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_CORE_INTERNAL_ACCOUNT_USER_DATA_FIXED_COUNTRY_CODE_USER_DATA_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_CORE_INTERNAL_ACCOUNT_USER_DATA_FIXED_COUNTRY_CODE_USER_DATA_H_
+
+#include "base/values.h"
+
+namespace brave_ads {
+
+base::Value::Dict BuildCountryCodeUserData();
```

Comment:
I think the advice about internal linkage also applies here. I understand that having this function separate makes testing it easier, but I suspect this functionality should be tested as part of `components/brave_ads/core/internal/account/confirmations/user_data_builder/fixed/confirmation_fixed_user_data_builder.cc`